ompi/mpi/fortran/use-mpi-f08/pmpi-f-interfaces-bind.h

```diff
@@ -98,9 +98,9 @@ end subroutine pompi_buffer_attach_f
 ! Note, too, we don't need a "p" version of the C implementation
 ! function -- Fortran's interfaces provide MPI_ and PMPI_ names for
 ! us; they can just both be bound to the same back-end
-! ompi_buffer_detach_f08 C function.
+! pompi_buffer_detach_f08 C function.
 subroutine pompi_buffer_detach_f(buffer_addr,size,ierror) &
-   BIND(C, name="ompi_buffer_detach_f08")
+   BIND(C, name="pompi_buffer_detach_f08")
```

Comment:
Ah, I was unaware of the Fortran restriction that we can't use the same binding name across multiple subroutines.  Bummer.

Ok. Can you update the comment in pmpi-f-interfaces-bind.h, and then add an implementation of `pompi_buffer_detatch_f08()`?  You can likely just add a 1-line routine in `ompi/mpi/fortran/use-mpi-f08/buffer_detatch.c` that turns around and calls `pompi_buffer_detatch_f08()` (i.e., just so that we can have 2 different C symbols for Fortran to call, but only have the body of the C routine once).